src/Window.vala

```diff
@@ -33,6 +33,7 @@ public class Akira.Window : Gtk.ApplicationWindow {
     public Akira.Utils.Dialogs dialogs;
 
     public Akira.StateManagers.CoordinatesManager coords_manager;
+    public Akira.StateManagers.SizeManager size_manager;
```

Comment:
what managers go in the window and which go in the canvas is really unclear--and was very confusing to me.

Reply:
That depends if you want the methods of those managers to affect the whole application and all its opened instances (windows) or if the methods should affect only the currently focused window.

Reply:
That depends on what the methods of those managers do.
The Size manager and coordinates manager affect and interact with the Transform Panel, which is a UI element outside the Canvas.
The Export Manager, or Nob manager, don't interact with any part of the UI other than the items in the Canvas, therefore they're bounded to the Canvas.
I'm more than happy to implement a better and more consistent approach.